### PR TITLE
[lte][agw] Changing py code to support 3.8 as well

### LINKED ISSUE #3842: Update AGW services to use python 3.8

### PR DESCRIPTION
**Pull request by AyliD**
Signed-off-by: Ayelet Regev Dabah <ayelet.regev@gmail.com>

## Summary

so this code can be run on ubuntu 20.04 

## Test Plan

with these changes tested that py3.5 integration tests will succeed as well as unitests (make test_all) 
on ubuntu 20.04 was able to test only the unitests part till full migration to py 3.8

## Additional Information

- [ ] This change is backwards-breaking

### CODE REVIEW

**Comment by 119Vik** [comment]
RestartCWAGTests

**Comment by 119Vik** [comment]
RestartCWAGTests

File: lte/gateway/python/magma/subscriberdb/tests/protocols/diameter/common.py [hunk 1]
```diff
@@ -0,0 +1,26 @@
+import asyncio
+# pylint: disable=W0223
+
+# We cannot create instances directly for transport we need them to be created by event loop so hacking this.
```
Comment (tmdzk) [inline lte/gateway/python/magma/subscriberdb/tests/protocols/diameter/common.py:4]: Pretty cool hack

**Review by tmdzk — APPROVED** [review]
This is perfect. I just want @uri200 to double check it

**Comment by 119Vik** [comment]
RestartCWAGTests

File: lte/gateway/python/magma/subscriberdb/tests/protocols/diameter/common.py [hunk 2]
```diff
@@ -0,0 +1,26 @@
+import asyncio
+# pylint: disable=W0223
+
```
Comment (uri200) [inline lte/gateway/python/magma/subscriberdb/tests/protocols/diameter/common.py:3]: maybe will use a more descriptive name for this file `MockTransport` would be fine.

**Review by uri200 — APPROVED** [review]

**Comment by 119Vik** [comment]
RestartCWAGTests

**Pull request merged**